scoresheets/CleanUp.tex

```diff
@@ -0,0 +1,25 @@
+
+The maximum time for this test is 5 minutes.
+
+\begin{scorelist}
+	\scoreheading{Main Goal}
+	\scoreitem[5]{100}{Place an object at the appropriate location}
+	\penaltyitem[5]{-30}{Receiving human help (telling where an object can be found)}
+	\penaltyitem[5]{-40}{Receiving human help (pointing out object to be moved)}
+	\penaltyitem[5]{-20}{Receiving human help (Guiding the robot near an object to be moved)}
+	\penaltyitem[5]{-30}{Receiving human help (telling robot which category an object is or where to place it)}
+	\penaltyitem[5]{-60}{Receiving human help (physically interacting with object)}
```

Comment:
I'm dubious about this last point. If a human operator moves the object and places it in the right spot, it might well cause a 100pts reduction (net to zero)

Reply:
Yes, I think the distribution penalties is odd.

If I point from across my living-room from the sofa, towards my desk, and say `tidy up that crap` and the robot does it and I haven't gotten off the sofa, then I think it's okay. Same with me saying `go clean up the kids toys from the spare bedroom` and again, I haven't gotten off the sofa, then I again think it's okay.

But they're all high, which means the teams are really pushed to solving the whole task as is so I think that is good.

Reply:
Let's leave the scoring at criteria of the referee. If a human solved the task, no points. If the human mostly solved the task but the robot somehow helped, it will score.